tree/ntuple/v7/src/RNTupleProcessor.cxx

```diff
@@ -155,6 +188,16 @@ ROOT::Experimental::NTupleSize_t ROOT::Experimental::RNTupleSingleProcessor::Loa
    return entryNumber;
 }
 
+void ROOT::Experimental::RNTupleSingleProcessor::SetEntryPointers(const REntry &entry)
+{
+   for (const auto &value : *fEntry) {
+      auto &field = value.GetField();
+      auto valuePtr = entry.GetPtr<void>(field.GetQualifiedFieldName());
```

Comment:
Not for this PR: but we should implement/find a way to iterate over all the pointers of an entry without all the field lookups.

Reply:
In principle this is offered through `REntry::ConstIterator_t` (and used on `fEntry` in l.193), no? In this specific case the field lookup is needed because a pointer from the entry *not* being iterated over is needed.